src/Native/System.Security.Cryptography.Native/pal_err.cpp

```diff
@@ -19,6 +19,11 @@ extern "C" uint64_t CryptoNative_ErrGetError()
     return ERR_get_error();
 }
 
+extern "C" uint64_t CryptoNative_ErrPeekError()
+{
+    return ERR_peek_error();
+}
+
 // TODO: temporarily keeping the un-prefixed signature of this method  
```

Comment:
Yes.  There's already a PR out to fix these TODOs, though, so I'm not adding the number.